addons/payment_odoo/models/payment_transaction.py

```diff
@@ -56,15 +56,17 @@ def _get_specific_rendering_values(self, processing_values):
             'shopperLocale': lang_code,
             'shopperReference': self.acquirer_id._odoo_compute_shopper_reference(
                 self.partner_id.id
-            ),
-            'recurringProcessingModel': 'CardOnFile',
+            ) if self.tokenize else '',
+            'recurringProcessingModel': 'Subscription',
```

Comment:
Why?

Reply:
Because Adyen :-D
The tokenization in PayByLink requires it to be `Subscription`

https://docs.adyen.com/online-payments/pay-by-link/api#tokenize-future-payments

Reply:
Ok for the model choice but it should still be `''` if the transaction is not tokenized, or Adyen will same payment details without the customer's consent.